internal/pkg/cli/validate.go

```diff
@@ -58,8 +58,8 @@ var ddbRegExp = regexp.MustCompile(`^[a-zA-Z0-9\-\.\_]+$`)
 
 // s3 validation expressions.
 // s3RegExp matches alphanumeric, .- from 3 to 63 characters long.
-// s3DashesRegExp matches consecutive dashes or periods
-// s3TrailingDashRegExp matches a trailing dash
+// punctuationRegExp matches consecutive dashes or periods
+// trailingPunctRegExp matches a trailing dash
 // ipAddressRegExp checks for a bucket in the format of an IP address.
 // https://docs.aws.amazon.com/awscloudtrail/latest/userguide/cloudtrail-s3-bucket-naming-requirements.html
```

Comment:
Seems like we have duplicated comments for explaining the same thing. Can we remove one of them?

Reply:
They look like they handle different things-- one is `--` and the other is `someName-`, right?

Reply:
Yes, that's exactly right